qiskit/primitives/base_sampler.py

```diff
@@ -91,12 +91,17 @@
 
 from abc import ABC, abstractmethod
 from collections.abc import Iterable, Sequence
+from copy import copy
```

Comment:
Can we update the example in the doc string above to use names for the circuits and also show passing circuits in some calls?

Reply:
Added in https://github.com/Qiskit/qiskit-terra/commit/5615215b006db600bbf592144d2e77b0e0c86248